README.md

```diff
@@ -158,43 +158,68 @@ Setting up and configuring of all these protections depends on and requires a lo
 
 ## Set up development environment
 
-### Local setup
+### Developing dependencies
+
+- API
+  - Required packages:
+    - `golang` >= 1.22
+    - `libexif` >= 1.15.1
+    - `dlib`
+    - `libjpeg`
+    - `libblas` 
+    - `libcblas`, recommending using `libatlas-base`
+    - `liblapack`
+  - Optional tools during developing:
+    - `reflex`
+    - `sqlite`
+- UI
+  - Required packages:
+    - `node` >= 18
+
+In Debian/Ubuntu, install dependencies:
 
-1. Install a local mysql server, and make a new database
-2. Rename `/api/example.env` to `.env` and update the `MYSQL_URL` field
-3. Rename `/ui/example.env` to `.env`
+```sh
+$ sudo apt install golang libexif-dev libdlib-dev libjpeg-dev libblas-dev libatlas-base-dev liblapack-dev libdlib-dev # For API requirement
+$ sudo apt install reflex sqlite3 # For API optional tools
+$ sudo apt install node # For UI
+```
 
-### Start API server
+In macOS, install dependencies:
 
-Make sure [golang](https://golang.org/) is installed.
+```sh
+$ brew install golang libheif dlib jpeg # For API
+$ brew install reflex sqlite3 # For API optional tools
+$ brew install node # For UI
+```
 
-Some C libraries are needed to compile the API, see [go-face requirements](https://github.com/Kagami/go-face#requirements) for more details.
```

Comment:
Why did you remove the Face-recognition dependency installation link? don't we need it anymore?

Reply:
They are already mentioned in the dependencies session. I make it more clear.